src/main/resources/application.yml

```diff
@@ -18,6 +18,10 @@ spring:
       test:
         - "stdout"
 
+  jpa:
```

Comment:
Does this mean we have to create the flyway migrations manually as SQL? Does JPA othewise try and create schemas from models? Not a problem, just for my own understanding.

Reply:
That's correct yeah - you can just let JPA do it itself but generally seen as too risky on a production database

Reply:
👍